EnglishFilter/sections/foreign.txt

```diff
@@ -3266,6 +3266,9 @@ gom.ge##a[href="http://topkinoebi.com/"]
 !---------------------------------------------------------------------
 !------------------------------ Hindi --------------------------------
 !---------------------------------------------------------------------
+! https://github.com/AdguardTeam/AdguardFilters/issues/79426
+hindikhabarwaala.com##img[width="400"][height="200"]
+hindikhabarwaala.com##img[width="400"][height="400"]
```

Comment:
Please add also these rules for ads on other pages:
```
hindikhabarwaala.com##img[style="width: 400px; height: 200px;"]
hindikhabarwaala.com##img[style="width: 400px; height: 250px;"]
hindikhabarwaala.com##img[style="width: 400px; height: 170px;"]
```

Reply:
Hm, too many rules. What about this `hindikhabarwaala.com##.post-content > div.post-text > p > img[src^="http://hindikhabarwaala.com/"][width="400"]`? It works ok on my end.

Reply:
That's better.